src/style.css

```diff
@@ -0,0 +1,75 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.blocks {
+  display: flex;
+  color: #fff;
+  font-size: 100px;
+  height: 300px;
+  min-width: 300px;
+  flex-grow: 1;
+  flex-basis: 33.3%;
+  justify-content: center;
+  align-items: center;
+}
+
+.first-block {
```

Comment:
the first version with n-th child was better, I meant to use class `.block` instead of using container